SwanPortAllocator/swanportallocator/portallocator.py

```diff
@@ -216,6 +228,10 @@ def send_error(content):
                             ports = self.get_ports(message['process'], message['n'])
                             send_ok(ports)
 
+                        elif message['action'] == Actions.RELEASE_PORT.value:
+                            self.release_ports(message['process'], message['ports'])
+                            send_ok(message['ports'])
```

Comment:
Why do you want to know the ports that have been removed (and therefore reply back with them)? If you do, maybe `release_ports` should return them (since they might not be the same as the ones sent).

Reply:
I guess it's for consistency, but I don't remember why bellow I sent the status back...

Reply:
Yes it was for consistency. As you say the status below is also sent and discarded.

Reply:
Should I add a commit that removes that parameter from `send_ok` both for set status and release ports?

Reply:
Yes, please

Reply:
For the actually released ports, we already issue a warning if we can't release a port because it does not belong to that process, I think that is enough and anyway, as you say, we won't do anything with those ports.